packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -204,16 +204,21 @@ const TreeView = React.forwardRef(function TreeView(props, ref) {
     setExpandedState(newExpanded);
   };
 
+  const isExpandable = (id) => {
```

Comment:
We should re-use this in TreeItem as well. https://github.com/mui-org/material-ui/blob/cbae0f8cf7b2426019973232e17f59d1da66420c/packages/material-ui-lab/src/TreeItem/TreeItem.js#L137